domains/yug.json

```diff
@@ -1,11 +1,12 @@
 {
+    "description": "My website",
+    "repo": "https://github.com/YUG38/YUG38.github.io",
     "owner": {
         "username": "YUG38",
-        "email": "4yugpatel123@gmail.com"
+        "email": "4yugpatel123@gmail.com",
+        "discord": "name_yug"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "YUG38.github.io"
```

Comment:
Redirects back to this domain for some reason. Invalid changes. No content is being displayed.

Reply:
how to solve?

Reply:
You have to fix your code on your github repository. URL should load a site, not redirect somewhere else in a endless loop.

Reply:
the website code is fine idk why it doesn't work when i try to use YUG38.github.io it redirects to yug.is-a.dev and if i try to use yug.is-a.dev then it won't load saying site not found

Reply:
Help pls?